XamarinCommunityToolkitSample/Pages/Views/CameraViewPage.xaml

```diff
@@ -69,7 +70,7 @@
 
                 <Button
                     x:Name="doCameraThings"
-                    Clicked="DoCameraThings_Clicked"
+                    Command="{Binding ShutterCommand}"
```

Comment:
```Command="{Binding ShutterCommand, Source={x:Reference cameraView}}"```

No need for ViewModel

Reply:
### ![](https://media1.giphy.com/media/75ZaxapnyMp2w/giphy.gif)